imagersite/imager_profile/forms.py

```diff
@@ -1,9 +1,30 @@
 from django import forms
 from .models import ImagerProfile
-# from django.contrib.auth.models import User
 
 
 class ProfileSettingsForm(forms.ModelForm):
+    first_name = forms.CharField(label='First Name', max_length=36)
+    last_name = forms.CharField(label='Last Name', max_length=36)
+    email = forms.EmailField(label='Email')
+
     class Meta:
         model = ImagerProfile
-        fields = ['address']
+        fields = ['address', 'fav_camera', 'photo_type', 'url']
+
+    def __init__(self, *args, **kwargs):
+        self.request = kwargs.pop("request")
+        super(ProfileSettingsForm, self).__init__(*args, **kwargs)
+        try:
+            self.fields['first_name'].initial = self.instance.user.first_name
+            self.fields['last_name'].initial = self.instance.user.last_name
+            self.fields['email'].initial = self.instance.user.email
```

Comment:
might be nice if the profile had a "kwargs for user form" method that provided a dictionary of these values.  It'd save you some lines of code here.